.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+## Title of the PR
+#### Trello/Jira board reference:
+* [Trello/Jira ticket #NUMBEROFTHETICKET](link goes here)
+---
+#### Description:
+* Add a description of what is the aim of this PR
+---
+#### Notes:
+*
+---
+#### Tasks:
+- [x] Add each element in this format
+---
+#### Risk:
+Options: (None, little, Medium, Risky)
+* Medium
```

Comment:
I'd remove this explanation and just leave for the dev to put in risk whatever they want to.